src/pages/eunjungPark/Main/Main.js

```diff
@@ -1,8 +1,271 @@
 import React, { Component } from 'react';
+import './Main.scss';
+import Nav from '../../../components/Nav/Nav';
+import Comment from './Comment';
+
+const commentList = [
+  {
+    id: 1,
+    userId: 'hoon-zz',
+    commentText: '은정아 라이언좀 그만 사',
+  },
+  {
+    id: 2,
+    userId: 'ryan.seoul.icon',
+    commentText: '춘식이는 프렌즈를 좋아해',
+  },
+  {
+    id: 3,
+    userId: 'boragom_molly',
+    commentText: '무더운 여름밤 춘식이와 나선 밤 산책',
+  },
+];
```

Comment:
- 상수는 상수임을 표기하기 위해서 대문자 스네이크 케이스로 작성해 주세요!
```suggestion
const COMMENT_LIST = [
  {
    id: 1,
    userId: 'hoon-zz',
    commentText: '은정아 라이언좀 그만 사',
  },
  {
    id: 2,
    userId: 'ryan.seoul.icon',
    commentText: '춘식이는 프렌즈를 좋아해',
  },
  {
    id: 3,
    userId: 'boragom_molly',
    commentText: '무더운 여름밤 춘식이와 나선 밤 산책',
  },
];
```